keyboards/h87a/keymaps/default/keymap.c

```diff
@@ -0,0 +1,39 @@
+#include "h87a.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	KEYMAP(
```

Comment:
This should be changed to `LAYOUT`, and needs to be fixed, as well (see the comment on the `h` file)